src/hooks/useConnectSovereignVenue.ts

```diff
@@ -0,0 +1,52 @@
+import { useMemo } from "react";
+
+import {
+  isSovereignVenueRequestedSelector,
+  sovereignVenueSelector,
+} from "utils/selectors";
+import { WithId, withId } from "utils/id";
+
+import { AnyVenue } from "types/venues";
```

Comment:
looks like we usually place `types/` earlier than `utils/`.
```suggestion
import { AnyVenue } from "types/venues";

import {
  isSovereignVenueRequestedSelector,
  sovereignVenueSelector,
} from "utils/selectors";
import { WithId, withId } from "utils/id";
```

Reply:
This hook will be removed and the new one will be used.
https://github.com/sparkletown/sparkle/pull/1725

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/f67996f11eaebb80d762933a33f1ecda0c64b057